index.html

```diff
@@ -0,0 +1,209 @@
+            <li>Back End Dev</li>
+            <li><img src="images/counter.png" alt=""/></li>
+            <li>End</li>
+          </ul>
+          <p class="parag">
+            A daily selection of privately personalized reads; no accounts
+            orsign-ups required.
+          </p>
+          <ul class="proj-tech">
+            <li><img src="images/htmli.png" alt="" /></li>
+            <li><img src="images/css.png" alt=""/></li>
+            <li><img src="images/javaScript.png" alt=""/></li>
+          </ul>
+          <img class="see-proj" src="images/see.png" alt="" />
+        </div>
+      </div>
+      <div class="project3">
+        <img src="images/portfolio.png" alt="" />
+        <div class="proj-desc">
+          <h2>Tonic</h2>
+          <ul class="list1">
+            <li>CANOPY</li>
+            <li><img src="images/counter.png" alt="" /></li>
+            <li>Back End Dev</li>
+            <li><img src="images/counter.png" alt=""/></li>
+            <li>End</li>
+          </ul>
+          <p class="parag">
+            A daily selection of privately personalized reads; no accounts
+            orsign-ups required.
+          </p>
+          <ul class="proj-tech">
+            <li><img src="images/htmli.png" alt=""/></li>
+            <li><img src="images/css.png" alt="" /></li>
+            <li><img src="images/javaScript.png" alt=""/></li>
+          </ul>
+          <img class="see-proj" src="images/see.png" alt=""/>
+        </div>
+      </div>
+      <div class="project4">
+        <img src="images/multi.png" alt="" />
+        <div class="proj-desc">
+          <h2>Multi-post <br>Story</h2>
+          <ul class="list1">
+            <li>CANOPY</li>
+            <li><img src="images/counter.png" alt=""/></li>
+            <li>Back End Dev</li>
+            <li><img src="images/counter.png" alt="" /></li>
+            <li>End</li>
+          </ul>
+          <p class="parag">
+            A daily selection of privately personalized reads; no accounts
+            orsign-ups required.
+          </p>
+          <ul class="proj-tech">
+            <li><img src="images/htmli.png" alt=""/></li>
+            <li><img src="images/css.png" alt=""/></li>
+            <li><img src="images/javaScript.png" alt="" /></li>
+          </ul>
+          <img class="see-proj" src="images/see.png" alt=""/>
+        </div>
+      </div>
+    </section>
+    <section class="about-sect">
+      <div class="about">
+        <h2 class="about-intro intro">About <br />Myself</h2>
+        <p class="about-pitch pitch">
+          I'm a software developer! I can help you build a product, feature or
+          website. Look through some of my work and experience! If you like what
+          you see and have a project you need coded, don't hesitate to contact
+          me.
+        </p>
+      </div>
+      <div class="connect conn-about">
+        <h3>LETS CONNECT</h3>
+        <ul class="social-icons">
```

Comment:
- [ ] Kindly select this ul in your css and add this code `font-size: x-large;
justify-content: space-evenly;
flex: fit-content;
margin-left: -30px; `they  gonna make your icons big and space between link this:
![Screenshot from 2022-05-28 19-23-24](https://user-images.githubusercontent.com/77842237/170833971-59f9962e-5497-479d-ab70-7edc8365d0aa.png)